### PR TITLE
Move Revision History to dots menu

### PR DESCRIPTION
**Pull request by MattBudz**
### Summary

Recent activity and revisions history show similar information. This PR consolidates the two and add's a `View History` link to the dots-menu.

### Check List

- [x] Added a CHANGELOG entry

### CODE REVIEW

File: spec/support/activity_shared_examples.rb [hunk 1]
```diff
@@ -54,32 +54,3 @@
     expect { submit_form }.not_to have_enqueued_job(ActivityTrackingJob)
   end
 end
-
-# Define the following let variables before using these examples:
-#
-#   create_activities : a block which creates the activities AND IS CALLED
-#                       BEFORE THE PAGE LOADS
-#   trackable: the model which the 'show' page is about
-shared_examples 'a page with an activity feed' do
```
Comment (aapomm) [inline spec/support/activity_shared_examples.rb:63]: I'm realizing that we still have this feed but it's now in the revisions view. In that case we still need to have a spec for that.

File: spec/features/revisions_spec.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe 'Revisions#show:' do
+  subject { page }
+
+  describe 'when the record has 2 revisions' do
+
```
Comment (aapomm) [inline spec/features/revisions_spec.rb:7]:
```suggestion
```

File: spec/features/revisions_spec.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    before do
+      login_to_project_as_user
+
+      with_versioning do
+        record.text = 'updated text'
+        record.save
+      end
+    end
+    
+    it 'the 2 revisions are listed in the history table' do
```
Comment (aapomm) [inline spec/features/revisions_spec.rb:23]:
```suggestion
    it 'lists the revisions in the history table' do
```

to make it make sense with the `it` block

File: spec/support/activity_shared_examples.rb (hunk 1 above)
Comment (MattBudz) [reply spec/support/activity_shared_examples.rb:63]: added specs

**Pull request merged**